addons/mail/models/mail_message.py

```diff
@@ -1142,9 +1143,7 @@ def _message_notification_format(self):
             'notifications': message.notification_ids._filtered_for_web_client()._notification_format(),
         } for message in self]
 
-    def _notify_message_notification_update(self):
-        """Send bus notifications to update status of notifications in the web
-        client. Purpose is to send the updated status per author."""
+    def _filter_can_read_record(self):
         messages = self.env['mail.message']
         for message in self:
             # Check if user has access to the record before displaying a notification about it.
```

Comment:
Comment seems too specific now. I'd add a docstring to the method and remove references to notifications in the docstring.

And let's keep the FIXME here I guess